Moving sharc task array data to decomisioned (#2211)
rcgsheffield/sheffield_hpc: hpc/scheduler/advanced_job_submission_and_control.rst

------------

This page details more advanced job submission and control methods that can be used 

> and the more modern :ref:`SLURM scheduler <slurm_info>` on Bessemer and Stanage.

Reviewer: ```suggestion
with the :ref:`SLURM scheduler <slurm_info>` on Bessemer and Stanage.
```
